docs/book/src/developer/testing.md

```diff
@@ -160,8 +160,21 @@ Execute the run configuration with `Debug`.
 
 <h1>Tips</h1>
 
-If you want to debug CAPI controller during e2e tests, just scale down the controller in the local kind cluster
```

Comment:
I dropped that wording because it's not that easy to just scale down the controller and run it via IDE (because of the webhook). The new tips make it easier by just connection against a controller deployed via tilt.